### ISSUE TITLE
Mehir is downgraded from commander of 2nd degree to 3rd degree

### ISSUE THREAD

**Issue by Slayer95** [inferno8/wesnoth-To_Lands_Unknown#22]
In S07 Dome of Elements, he's 2nd degree. By S12, he's 3rd degree.

AFAIK, military progression should go 3rd degree into 2nd degree into 1st degree.

**Comment by inferno8** [comment]
Well, in Summoners' society military progression could work in a different way, very similar to how unit level works (higher level = more power).

Actually this is very interesting. Nobody has ever complained about the current military progression in TLU, so at the moment I prefer to do nothing about it or to add a narrator note somewhere explaining how degrees work in the campaign to avoid potential confusion.
Reordering the numbers is also an option (the easiest one), but I need to think about it.

**Comment by Slayer95** [comment]
I'm fine with Summoner degree going upwards (1st into 2nd into 3rd into 4th). It's only the military side of it that feels odd. Although it's certainly an arbitrary convention.

It's also to be noted that in Dome of Elements, the instructor says Mehir had already acquired a new rank _before_ he gets his feet on his Grand Summoner personal circle. So it's not clear that there is a correlation or identity between his summoner and military degree.

**Comment by inferno8** [comment]
> it's not clear that there is a correlation or identity between his summoner and military degree.

There are actually two things: military ranks and summoner degrees. Take a look at this story fragment from S07 (after Mehir rescued the lizards):

> Because Mehir’s achievement was beyond exceptional for his lack of experience, Sharif promoted him by multiple ranks.

And this one is a fragment of Grand Summoner's description:

> Summoners, scholars and warriors alike will eventually reach the Second Degree provided enough talent and cunning. The Third Degree, however, does not come easily to non-geniuses of the arts. ‘Grand Summoners’, as they are called by their peers, value both war and wisdom equally in earning their precious knowledge of summoning elementals from the Abyss. Respected and feared, grand summoners eliminate foes with absolute certainty in the wake of their circles of destruction. They also no longer need to use scrolls for summoning.

A person doesn't have to serve in the army to obtain a specific summoner degree as it is more of an 'arts' thing, but many high-ranking military officers are summoners of third or fourth degree, because this makes them more useful on a battlefield + it is also about prestige.

Here's what the instructor says in S07:

>  You see, with your new rank, you are entitled to the privilege of owning a personal magic circle, the combat-oriented kind.

This indicates that there is a correlation with a military rank and summoner degree. Mehir is now a commander, so he can own a magic circle. Such circle is usually associated with summoners of the third degree, so we can assume Mehir got the promotion in the field of summoning along with his new military rank "out of the box".
The Grand Summoner's description says, one has to be very skilled at the art of summoning to actually become the summoner of the third degree, but in case of Mehir it was more about his tactical skills.
So in conclusion, if you want to become a Grand Summoner you have to study hard or make some military achievements like saving a bunch of lizards in a goddamned canyon. But even then you will have to train hard to be able to understand and effectively use new tools like the aforementioned personal circle.

And having said that, I guess this line of dialogue (07_Dome_of_Elements.cfg:127) probably needs to be changed:
Mehir: _That's right. Mehir, commander of the second degree, in person! And who might you be?_
to this:
Mehir: _That's right. Mehir, commander and now summoner of the third degree, in person! And who might you be?_
